app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,24 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_tag url: '/login', layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-4" do |f|
+        %form.sign_in
```

Comment:
In order to pass `.sign_in` to `bootstrap_form_for` you need to use `html: { class: "sign_in }` or something similar. Study the Rails API docks for form_for and bootstraps_form docs too.